fabric/bin/peer.sh

```diff
@@ -36,15 +36,19 @@ handle_chaincode_install() {
 		CC_LANG=$2
 		shift;shift
 		;;
+        -e|--enclavesopath)
```

Comment:
Why add a new parameter? This is really semantically the -p|--path option for language fpc-c?

Reply:
it was not clear at all that -p was there for this.
anyway, fixed